README.md

```diff
@@ -1,3 +1,17 @@
+# Hierarchical deterministic password manager
+
+## Compilation Instructions
```

Comment:
Should go below `## Requirements` section in readme

Reply:
This was mostly caused by the fact that when I forked there was no readme.